server/tools/x509.sh

```diff
@@ -25,6 +25,8 @@ function autogenerate_keystores() {
 
       echo "Creating ${KEYSTORES[$KEYSTORE_TYPE]} keystore via OpenShift's service serving x509 certificate secrets.."
 
+      rm -f "${JKS_KEYSTORE_FILE}" "${PKCS12_KEYSTORE_FILE}"
```

Comment:
Should really just check if this was already done instead of re-doing it

Reply:
It's not trivial, because existing keystores were created with random passwords. These passwords are saved in standalone-ha.xml, but reading them from it looks too complex. And handling changes of certs and keys would add even more complexity.

Reply:
Running this whole script should be conditional as well. If it has been ran, do not do it again.

Reply:
@slaskawi 
How certificate is supposed to be updated then?